enhanced-consent-accelerator/streamlit/pages/1_CMP_Dashboard.py

```diff
@@ -17,23 +26,27 @@
 c1, c2, c3 = st.columns((8, 1, 8))
 
 with c1:
-    fig = px.scatter(df_cmp_stats, x='CMP load time', y='First consent event type', width=700, height=400, title="<b> CMP load time vs consent choice<b>")
+    data['cmp_stats'] = data['cmp_stats'].rename(columns = {'cmp_load_time' : 'CMP load time',
+                                                            'first_consent_event_type': 'First consent event type',
+                                                            'cmp_interaction_time': 'CMP interaction time'
+                                                            })
+    fig = px.scatter(data['cmp_stats'], x='CMP load time', y='First consent event type', width=700, height=400, title="<b> CMP load time vs consent choice<b>")
```

Comment:
```suggestion
    fig = px.scatter(data['cmp_stats'], x='cmp load time', y='first consent event type', width=700, height=400, title="<b> CMP load time vs consent choice<b>")
```
Not sure why, but it didn't work with capitalised x & y

Reply:
My bad, I had saved an older version of the data with column names with spaces in (which bigquery does not allow), it should work now?

Reply:
I'm using snowflake, would BQ still break it? It just tried it again and still wasn't working. Same error as before

Reply:
I didn't push my commit, will do it in the morning 😅